web/src/models/TestRunResult.model.ts

```diff
@@ -1,10 +1,41 @@
+import {differenceInSeconds} from 'date-fns';
+import {TAssertionResultList} from '../types/Assertion.types';
 import {IRawTestRunResult, ITestRunResult} from '../types/TestRunResult.types';
 import Trace from './Trace.model';
 
+const getExecutionTime = (createdAt?: string, completedAt?: string) => {
```

Comment:
Adding some of the calculations at the model level instead of making them during the component render